doc/build.md

```diff
@@ -14,63 +14,56 @@ for auto-completion.
 
 ## Building in Container
 
-We build a Docker image that contains development tools:
+We build a Docker image that contains development tools below. Because this repo contains Go code, please make sure that you have the directory
+structure required by Go. 
```

Comment:
> "Because this repo contains Go code, please make sure that you have the directory structure required by Go. "

This sentence should come after the bullet points.

Reply:
Fixed.